### ISSUE TITLE
[1.20.1 Forge] restarts up to 3 times, popup notification for user to restart client upon an launch sequence update case not showing up

### ISSUE THREAD

**Issue by MrKashew** [Skidamek/AutoModpack#144]
### Motivation

For users not viewing Console such as
[20:27:33] [modloading-worker-0/INFO] [AutoModpack/]: Restart your client!

### Description

For this case some sort of UX notification
[24/07/2023 22:39:23 PM] [modloading-worker-0/INFO] Restart your client!

**Comment by Skidamek** [comment]
There is actually maybe it doesn't shows on forge somehow

**Comment by MrKashew** [comment]
yep this is on Forge 1.20.1

**Comment by Nosonii** [comment]
[Minecraft 1.20.1 AutoModpack 3.4.3 Forge]

I am having this problem too, after I synced my client to my server and restarted, every time I launch the game AutoModpack will force close the game after all the mods are loaded with exit code 0, and when disabling AutoModpack the game will be able to launch.

I suspected this may have been due to a mismatch of having client-side mods that are not present in the server's mod directory, so I made the directory /automodpack/host-modpack/mods/ and put ALL the mods in there but the force closing issue is still there after I synced again.

**Comment by Skidamek** [comment]
@Nosonii oh what, so that's a different not releated issue, please open new issue

**Comment by MrKashew** [comment]
also is this flow expected or is this part of bug
start client -> client updates, force close (no popup) -> restart client again ->  client logs same updates, force close (no popup)  -> restart client (success)

**Comment by Skidamek** [comment]
> is this flow expected or is this part of bug
> start client -> client updates, force close (no popup) -> restart client again ->  client logs same updates, force close (no popup)  -> restart client (success)

Definitely no it should be like that:

start client -> client updates -> show popup -> popup button click -> force close game -> start game successfully

**Comment by Skidamek** [comment]
I guess it's the issue because forge loads mods in parallel

**Comment by MrKashew** [comment]
gotcha sum of bugs here for future reference:

- forge has weird behavior where if update happens in launch:
    - start client -> client updates, force close (no popup) -> restart client again -> client logs same updates, force close (no popup) -> restart client (success)
    - should be start client -> client updates -> show popup -> click button on popup -> force close game -> manually start game agian successfully
-  pop up does not show on force close
- forge does not have the toast messages as seen in video

**Comment by Skidamek** [comment]
It may fix this:
https://github.com/Skidamek/AutoModpack/suites/14637674445/artifacts/830252211

**Comment by Skidamek** [comment]
Fixed in 3.4.4
